src/Files.App/Dialogs/ReorderSidebarItemsDialog.xaml

```diff
@@ -0,0 +1,84 @@
+﻿<ContentDialog
+	x:Class="Files.App.Dialogs.ReorderSidebarItemsDialog"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:navigationcontrolitems="using:Files.App.DataModels.NavigationControlItems"
+	Title="{helpers:ResourceString Name=ReorderSidebarItemsDialogText}"
+	DefaultButton="Primary"
+	IsPrimaryButtonEnabled="True"
+	PrimaryButtonCommand="{x:Bind ViewModel.PrimaryButtonCommand}"
+	PrimaryButtonText="{helpers:ResourceString Name=Save}"
+	RequestedTheme="{x:Bind helpers:ThemeHelper.RootTheme}"
+	SecondaryButtonText="{helpers:ResourceString Name=Cancel}"
+	Style="{StaticResource DefaultContentDialogStyle}"
+	mc:Ignorable="d">
+
+	<Grid
+		x:Name="DestinationPathGrid"
+		ColumnSpacing="8"
+		RowSpacing="8">
+		<Grid.ColumnDefinitions>
+			<ColumnDefinition />
+			<ColumnDefinition Width="Auto" />
+		</Grid.ColumnDefinitions>
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition Height="Auto" />
+			<RowDefinition Height="Auto" />
+		</Grid.RowDefinitions>
+
+		<NavigationView
```

Comment:
NavigationView is a pretty heavy control, can you see if ListView works instead?

Reply:
List view works, but I couldn't find a way to use the icon without using NavigationViewItems as children (and they caused problems with the selection box size iirc)

Reply:
I can't get it to work with listview, the DataContext is breaking.

Reply:
We need a new DataContext.

Reply:
How would that be done?

Reply:
What do you mean by DataContext is breaking?

Reply:
It throws a StackOverflow when the dialog is created

Reply:
Can you push the commit anyways? I'll see if I can reproduce and debug the issue.

Reply:
Ok

Reply:
It's probably me being stupid but I can't figure it out